src/util/mathematical_expr.h

```diff
@@ -204,6 +207,9 @@ class function_application_exprt : public binary_exprt
     return op0();
   }
 
+  /// the type of the expression returned by \ref function
```

Comment:
I'm not sure we are perfectly consistent about this, but I'd think we use a full sentence here (i.e., uppercase the first word and end with a full stop.).